examples/base/src/index.ts

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
Add comment that this is just example TypeScript file. 
It may be confusing for someone.